.github/workflows/chromatic-main-and-prs.yml

```diff
@@ -15,7 +15,7 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: lts/*
```

Comment:
What are pros and cons to using LTS tag vs pinning the version? My biggest concern is that this tag can change the major versions when we may not be ready to.

Reply:
That's the point. If it fails, we'll know early on (before customers run into it).

Reply:
But how do we know any any changes we made to support new version still work on existing node version?

Reply:
Well I guess that's mitigated by saying we are dropping support for the older versions. I guess we would need to be okay with the small windows where multiple versions are still supported

Reply:
I discussed with Paul that we should support whatever version is still being maintained (i.e. isn't end-of-life) according to the [Node.js release schedule](https://github.com/nodejs/release#release-schedule). Right now that's Node 18+. We'll test against each LTS version (even numbers) as well as the latest.

For most tests the exact Node version doesn't matter, and so I'm specifying `lts/*` for those so that we won't have to update them over time. For some tests it _does_ matter, and those specify something different. The LTS versions are covered in `smoke-test-node18.yml` and `smoke-test-node20.yml`, while the latest (currently Node 21) is covered in `smoke-test-node-api.yml`.

Reply:
That sounds good. As long as we mitigate the tag changing and catching us unaware, I am fine!